src/modules/search/hooks/useSearch.test.js

```diff
@@ -0,0 +1,93 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { act } from 'react-dom/test-utils';
+import { useCustomQuery } from 'src/modules/common/hooks';
+import { queryWrapper as wrapper } from 'src/utils/test/queryWrapper';
+import { useSearch } from './useSearch';
+
+jest.mock('src/modules/common/hooks', () => ({
+  useCustomQuery: jest.fn().mockImplementation(({ options }) => {
+    if (!options.enabled) {
+      return {};
+    }
+    return {
+      data: { data: [{ id: '1' }] }
+    }
+  })
+}))
+
+
+describe('useSearch hook', () => {
+
+  it('should not call api for invalid search value', () => {
+    act(() => {
+      const { result, waitFor } = renderHook(() => useSearch(), { wrapper });
+      waitFor(() => !result.current.isLoading);
+      expect(result.current.delegates.length).toEqual(0)
+
+    })
+    // Search value with length less than three
+    act(() => {
+      const { result, waitFor } = renderHook(() => useSearch('hi'), { wrapper });
+      waitFor(() => !result.current.isLoading);
+      expect(result.current.delegates.length).toEqual(0)
+    })
```

Comment:
I think this should have its own separate test case instead of it been here and identifying it with a comment